x-pack/solutions/observability/plugins/observability/public/pages/cases/components/cases.tsx

```diff
@@ -109,3 +91,74 @@ export function Cases({ permissions }: CasesProps) {
     </>
   );
 }
+
+function InvertedAlertsTable({ props }: { props: CaseViewAlertsTableProps }) {
```

Comment:
This is a small wrapper around the ObservabilityAlertsTable that handles the inversion I'm trying to demo here. It lets me handle the modal outside the table and define the callbacks where I have access to the alert and case in scope, outside of the alerts table component. 

PHEW, what a terrible adventure.